ARDOPC/Makefile

```diff
@@ -1,13 +1,72 @@
 #	ardopcf Makefile
+#		For Linux, the default build requires only gcc and libraries typically
+#		installed by default.
+#			cd ardop/ARDOPC
+#			make
+#
+#		Fow Windows, the default build requires installation of a MinGW build
+#		environment.  The easily installable packages from https://winlibs.com
+#		available for 32-bit or 64-bit builds are suggested.  These are used to
+#		build the Windows releases.  Installing these in `C:\Program Files` is
+#		not recommeneded since that may require admin privileges.  Other build
```

Comment:
If we recommend using the system path on Linux, we should consider doing the same on Windows. This is ultimately easier for the casual dev, IMO, as you don't have to worry about the runtime search path for the compiler.

Reply:
In my experience with Windows anything requiring admin privileges while using command line tools is much less convenient than on Linux where `sudo` can be used.